charts/datadog/ci/kubeval.yaml

```diff
@@ -2,10 +2,8 @@ datadog:
   apiKey: "00000000000000000000000000000000"
   appKey: "0000000000000000000000000000000000000000"
   kubelet:
-    host:
-      valueFrom:
-        fieldRef:
-          fieldPath: status.hostIP
```

Comment:
did that means we don't need to test it anymore?

Reply:
We now test that the default behaviour (which is to use `status.hostIP`) works properly